ketcher-autotests/tests/Structure-Creating-&-Editing/Text-Tool/text-formating.spec.ts

```diff
@@ -0,0 +1,254 @@
+  });
+
+  test.afterEach(async ({ page }) => {
+    await takeEditorScreenshot(page);
+  });
+
+  test('Text tool - Font size', async ({ page }) => {
+    // Test case:EPMLSOPKET-2885
+    // Verify if possible is changing font size on the created text object
+    await addTextBoxToCanvas(page);
+    await page.getByRole('dialog').getByRole('textbox').fill('TEST');
+    await page.keyboard.press('Control+a');
+    await page.getByRole('button', { name: '13' }).click();
+    await page.getByText('20', { exact: true }).click();
+    await pressButton(page, 'Apply');
+    await takeEditorScreenshot(page);
+    await page.getByText('TEST').dblclick();
+    await page.keyboard.press('Control+a');
+    await page.getByRole('button', { name: '20' }).click();
+    await page.getByText('10', { exact: true }).click();
+    await pressButton(page, 'Apply');
+  });
+
+  test('Text tool - Applying styles - Bold', async ({ page }) => {
+    // Test case: EPMLSOPKET-2256
+    // Verify if possible to put bold style on the created text object
+    await addTextBoxToCanvas(page);
+    await page.getByRole('dialog').getByRole('textbox').fill('ABC');
+    await page.keyboard.press('Control+a');
+    await page.getByRole('button', { name: 'bold' }).click();
+    await pressButton(page, 'Apply');
+    await takeEditorScreenshot(page);
+    await page.getByText('ABC').dblclick();
+    await page.keyboard.press('Control+a');
+    await page.getByRole('button', { name: 'bold' }).click();
+    await page.getByRole('button', { name: 'bold' }).click();
+  });
+
+  test('Text tool - Applying styles - Italic', async ({ page }) => {
+    // Test case: EPMLSOPKET-2257
+    // Verify if possible to put Italic style on the created text object
+    await addTextBoxToCanvas(page);
+    await page.getByRole('dialog').getByRole('textbox').fill('ABCDE');
+    await page.keyboard.press('Control+a');
+    await page.getByRole('button', { name: 'italic' }).click();
+    await pressButton(page, 'Apply');
+    await takeEditorScreenshot(page);
+    await page.getByText('ABCDE').dblclick();
+    await page.keyboard.press('Control+a');
+    await page.getByRole('button', { name: 'italic' }).click();
+    await page.getByRole('button', { name: 'italic' }).click();
+  });
+
+  test('Text tool - Applying styles - Subscript', async ({ page }) => {
+    // Test case: EPMLSOPKET-2258
+    // Verify if possible to put Subscript style on the created text object
+    await addTextBoxToCanvas(page);
+    await page.getByRole('dialog').getByRole('textbox').fill('ABC123');
+    await page.keyboard.press('Control+a');
+    await page.getByRole('button', { name: 'subscript' }).click();
+    await pressButton(page, 'Apply');
+    await takeEditorScreenshot(page);
+    await page.getByText('ABC123').dblclick();
+    await page.keyboard.press('Control+a');
+    await page.getByRole('button', { name: 'subscript' }).click();
+    await page.getByRole('button', { name: 'subscript' }).click();
+  });
```

Comment:
I propose modify it like example:
async function applySubscript(page:Page, text:string, buttonName:string) {
  await page.getByRole('dialog').getByRole('textbox').fill(text);
  await page.keyboard.press('Control+a');
  await page.getByRole('button', { name: buttonName }).click();
  await pressButton(page, 'Apply');
  await takeEditorScreenshot(page);
  await page.getByText(text).dblclick();
  await page.keyboard.press('Control+a');
  await page.getByRole('button', { name: buttonName }).click();
  await page.getByRole('button', { name: buttonName }).click();
}

Reply:
and then only use await applySubscript(page, 'ABC123', 'subscript');

Reply:
I suggest using this approach in all autotests written here

Reply:
done